jetty-core/jetty-util/src/test/java/org/eclipse/jetty/util/resource/ResourceTest.java

```diff
@@ -343,6 +343,17 @@ public void testEqualsWindowsCaseInsensitiveDrive() throws Exception
         assertEquals(rb, ra);
     }
 
+    @Test
+    public void testResourceExtraSlashStripping() throws Exception
+    {
+        Resource ra = Resource.newResource("file:/a/b/c");
```

Comment:
Oops, sorry, you are using the original (wrong) version in my comments.

See: https://github.com/eclipse/jetty.project/pull/8106#discussion_r914832327

Reply:
Let's test both.